scripts/us_census/pep_srh/mcf_generator.py

```diff
@@ -0,0 +1,99 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module generates MCF file in OUTPUT directory.
+There are two mcf files generated
+1. population_estimate_by_srh.mcf - for importing as-is data from US Census
+2. population_estimate_by_srh_agg.mcf - for importing aggregated data
+"""
+
+import os
+from constants import OUTPUT_DIR, MCF_SRH, MCF_SH, MCF_RH
+
+_CODEDIR = os.path.dirname(os.path.realpath(__file__))
+
+
+def generate_mcf():
+    """
+    This function generates MCF file in OUTPUT directory.
+    There are two mcf files generated
+    1. population_estimate_by_srh.mcf - for importing as-is data from US Census
+    2. population_estimate_by_srh_agg.mcf - for importing aggregated data
+    OUTPUT directory is used as common between process.py and process_test.py
+    """
+    os.system("mkdir -p " + _CODEDIR + OUTPUT_DIR)
+    genderType = ['Male', 'Female']
+    hispanic = ['HispanicOrLatino', 'NotHispanicOrLatino']
+    race = [
+        'WhiteAlone', 'BlackOrAfricanAmericanAlone',
+        'AmericanIndianOrAlaskaNativeAlone', 'AsianOrPacificIslander',
+        'AsianAlone', 'NativeHawaiianOrOtherPacificIslanderAlone',
+        'TwoOrMoreRaces', 'WhiteAloneOrInCombinationWithOneOrMoreOtherRaces',
+        'BlackOrAfricanAmericanAloneOrInCombinationWithOneOrMoreOtherRaces',
+        'AmericanIndianAndAlaskaNativeAloneOrInCombinationWithOneOrMoreOtherRaces',
+        'AsianAloneOrInCombinationWithOneOrMoreOtherRaces',
+        'NativeHawaiianAndOtherPacificIslanderAloneOrInCombination\
+WithOneOrMoreOtherRaces'
+    ]
+
+    # Count_Person_Male_NotHispanicOrLatino_WhiteAlone
+    # Count_Person_Female_NotHispanicOrLatino_WhiteAlone
+    # SV's are already present, hence not generating them
+    mcf = ""
+
+    for gt in genderType:
+        for h in hispanic:
+            for r in race:
+                if 'Count_Person_' + gt + '_' + h + '_' + r not in [
+                        'Count_Person_Male_NotHispanicOrLatino_WhiteAlone',
+                        'Count_Person_Female_NotHispanicOrLatino_WhiteAlone'
+                ]:
+                    mcf += MCF_SRH.format(node_name='Count_Person_' + gt + '_' +
+                                          h + '_' + r,
+                                          gender=gt,
+                                          race=h + '__' + r)
+
+    for gt in genderType:
+        for h in hispanic:
+            mcf += MCF_SRH.format(node_name='Count_Person_' + gt + '_' + h,
+                                  gender=gt,
+                                  race=h)
+
```

Comment:
Pls add a comment indicating why the template: MCF_RH is not used fro statvar generation here.